### PR TITLE
Fix: Cannot add new API Revisions for APIs with MSSQL as Database

### PR DESCRIPTION
**Pull request by BLasan**
**Fixes**
- https://github.com/wso2/api-manager/issues/1318

### CODE REVIEW

**Comment by rmsamitha** [comment]
Please check if this would fail for other DB types

**Comment by BLasan** [comment]
Fix is working for `mssql` when deploying an API product or an API

**Comment by BLasan** [comment]
Please don't merge. Will update the PR now

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java [hunk 1]
```diff
@@ -16428,6 +16428,8 @@ public void addAPIRevision(APIRevision apiRevision) throws APIManagementExceptio
                 String driverName = connection.getMetaData().getDriverName();
                 if (driverName.contains("Oracle")) {
                     getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES_ORACLE_SQL;
+                } else if (driverName.contains("Microsoft JDBC")) {
+                    getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES_MSSQL;
```
Comment (rmsamitha) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java:16432]: Don't we need to mention as below: ?

https://github.com/wso2/carbon-apimgt/blob/dc0af27d72fcc1c9c63b056266ff06e221ee2dca/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L16236-L16237

**Review by rmsamitha — APPROVED** [review]

**Pull request merged**